samples/boards/st/uart/single_wire/README.rst

```diff
@@ -12,6 +12,8 @@ functionality of STM32. Without adaptions this example runs on STM32F3 discovery
 board. You need to establish a physical connection between pins PA2 (USART2_TX) and
 PC10 (UART4_TX).
 
+To support the `nucleo_wba55cg` board, establish a physical connection between pins PA12 (USART2_TX) and PA2 (LPUART1_TX).
+
```

Comment:
I would club this into a single description pushing readers to go to each board overlay to see which connection should be made, in order to avoid the need to modify this each time a new board is added